lib/addons/karpenter/index.ts

```diff
@@ -280,7 +280,7 @@ export class KarpenterAddOn extends HelmAddOn {
                 clusterEndpoint: endpoint,
                 clusterName: name,
                 defaultInstanceProfile: karpenterInstanceProfile.instanceProfileName,
-                interruptionQueueName: stackName
+                interruptionQueueName: interruption ? stackName : "",
```

Comment:
If interruptionQueueName is empty string, is the same as not providing the interruptionQueueName attribute at all? Was it tested?

Reply:
yes - the default value is "" as shown [here](https://github.com/aws/karpenter/blob/5e02e4c6eb3432b5da12df100474a114db66607f/charts/karpenter/values.yaml#L187)